yb-voyager/cmd/analyzeSchema.go

```diff
@@ -1103,7 +1103,10 @@ func packAndSendAnalyzeSchemaPayload(status string) {
 	if !callhome.SendDiagnostics {
 		return
 	}
-	payload := createCallhomePayload()
+	payload, shouldSend := createCallhomePayload()
```

Comment:
It seems wrong for `createCallhomePayload` to return a value that tells you whether you should send the payload or not. `createCallhomePayload` should only deal with creating the payload. 

How about:
- define another function `shouldSendCallhome()` which returns false if either callhome.sendDiagnostics is false OR startTime is uninitialized. 
- call that function on line 1103.

Reply:
Also add a comment explaining that startTime being uninitialized is a proxy for detecting that export-dir wasn't set up properly.

Reply:
Yeah makes sense.